### PR TITLE
Set CadaverSizeTarget to 0 for Scenario testing

### PR DESCRIPTION
**Pull request by nicholasguoalgorand**
## Summary

Set CadaverSizeTarget to 0 so that we don't suffer huge performance losses in tests

## Test Plan

How did you test these changes? Please provide the exact scenarios you tested in as much detail as possible including commands, output and rationale.

### CODE REVIEW

File: test/testdata/deployednettemplates/recipes/scenario1/node.json [hunk 1]
```diff
@@ -5,18 +5,18 @@
...
-        "APIToken": "{{APIToken}}",
-        "EnableBlockStats": true,
-        "EnableTelemetry": true,
-        "TelemetryURI": "{{TelemetryURI}}",
-        "EnableMetrics": true,
-        "MetricsURI": "{{MetricsURI}}",
-        "ConfigJSONOverride": "{  \"TxPoolExponentialIncreaseFactor\": 1, \"DNSBootstrapID\": \"<network>.algodev.network\", \"DeadlockDetection\": -1, \"PeerPingPeriodSeconds\": 30, \"EnableAgreementReporting\": true, \"EnableAgreementTimeMetrics\": true, \"EnableAssembleStats\": true, \"EnableProcessBlockStats\": true, \"BaseLoggerDebugLevel\": 4, \"EnableProfiler\": true }",
-        "FractionApply": 0.2
-    }
+    "ConfigJSONOverride": "{  \"TxPoolExponentialIncreaseFactor\": 1, \"DNSBootstrapID\": \"<network>.algodev.network\", \"DeadlockDetection\": -1, \"PeerPingPeriodSeconds\": 30, \"BaseLoggerDebugLevel\": 4, \"CadaverSizeTarget\": 0  }",
```
Comment (tsachiherman) [inline test/testdata/deployednettemplates/recipes/scenario1/node.json:8]: is there any specific reason you removed the `EnableProfiler` flag ?

**Review by tsachiherman — CHANGES_REQUESTED** [review]
only one comment regarding the EnableProfiler on scenario 1; also, could you confirm that the mainnet-model didn't require that because it was already using 0 CadaverSizeTarget ?

**Review by egieseke — APPROVED** [review]
Looks OK.

File: test/testdata/deployednettemplates/recipes/scenario1/node.json (hunk 1 above)
Comment (nicholasguoalgorand) [reply test/testdata/deployednettemplates/recipes/scenario1/node.json:8]: I removed it because it wasn't in the net.json file, i'll add it back

**Review by tsachiherman — APPROVED** [review]

**Pull request merged**